WcaOnRails/.eslintrc.json

```diff
@@ -25,10 +25,11 @@
         "jquery"
     ],
     "rules": {
```

Comment:
What are these changes about? Do they have anything to do with your changes to the cookie banner?

Reply:
Sort of. It's part related to the `console.warn` I added, and part unrelated cleanup. Please see the last paragraph of my commit message ("Sorry for the unrelated seeming changes in .eslintrc.json...") for a full explanation.

Reply:
Whoops, my bad for missing that part of your PR description.

Reply:
Seeing that the previous configuration was set in `application.js` as
```js
/* eslint no-console:0 */
```
I wonder how you came up with this new
```js
"no-console": ["error", { "allow": ["warn", "error"] }]
```
config value. Can you please explain what the `allow` block even does and why it contains `error`? To me this reads a bit like "no console for errors but allow errors"?!

Reply:
It also seems reasonable to me, thanks for the thorough explanation!
Also I totally share your opinion about string values being more concise and better readable than their `0|1|2` int counterparts